scripts/cli/package.json

```diff
@@ -1,9 +1,23 @@
 {
-  "name": "pmesh-stats",
+  "name": "polymesh-cli",
   "version": "1.0.0",
-  "description": "Polymesh stats collector",
-  "main": "index.js",
+  "description": "CLI for Polymesh",
+  "main": "/dist/polymesh_cli.js",
+  "bin": {
+    "polymesh-cli": "./dist/polymesh_cli.js"
+  },
   "scripts": {
+    "start": "npm run build && npm run global",
+    "create": "npm run build && npm run test",
+    "build": "npm run clean:some && tsc -p .",
+    "global": "sudo npm i -g && polymesh-cli",
```

Comment:
It is possible to assign a user directory to the `-g` flag: https://github.com/sindresorhus/guides/blob/master/npm-global-without-sudo.md. If I already have this configuration, running with sudo will screw up ownership of my npm directory.

Since the above modification of `npm -g` depends on additional local configuration, to keep the script self-contained it may be better to use nvm. Another advantage of nvm is the ability to specify the exact versions of npm and node.

Reply:
I have made the changes so sudo is no longer needed though on the nvm topic it seems I can't use it to install packages only like you said to change the versions of npm and node.

Reply:
You can also run the cli with node ./dist/polymesh_cli.js [commands] [args] which avoids having a clash with node_modules and global packages.